pkg/expression/builtin_time_test.go

```diff
@@ -874,6 +874,7 @@ func TestNowAndUTCTimestamp(t *testing.T) {
 	// Test that "timestamp" and "time_zone" variable may affect the result of Now() builtin function.
 	err := ctx.GetSessionVars().SetSystemVar("time_zone", "+00:00")
 	require.NoError(t, err)
+	ctx.GetSessionVars().StmtCtx.SetTimeZone(ctx.GetSessionVars().TimeZone)
```

Comment:
Is it redundant to `SetSystemVar()` and immediately `GetSystemVar()` for `"time_zone"`? I mean, why not just using `ctx.GetSessionVars().StmtCtx.SetTimeZone("+00:00")`?

Reply:
Because "+00:00" is not a `*time.Location` format.

Reply:
> Because "+00:00" is not a `*time.Location` format.

I don't think format is important... why so not just using `ctx.GetSessionVars().StmtCtx.SetTimeZone(time.UTC)`?